src/ConcatenateMsg.chpl

```diff
@@ -115,7 +115,7 @@ module ConcatenateMsg
                 }
                 var repMsg = "created " + st.attrib(segName) + "+created " + st.attrib(valName);
                 cmLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),
-                                  "created concatenated pdarray %t".format(st.lookup(valName)));
+                                  "created concatenated pdarray %s".format(valName));
```

Comment:
could we change this to st.attrib(valName)? That will provide the info I think would be good for debugging purposes.

Reply:
Yup, sounds good. I figured just printing `name` wasn't the right thing to do, but it was a quick fix. Happy to change to `st.attrib()`

Reply:
@ronawho Cool, yeah, like I wrote, I changed some of the st.lookup to st.attrib once I'd worked out some debugging stuff for another issue, but I missed the ones you found. Again, great work figuring this out!

Reply:
Changed